bundles/org.openhab.binding.visualcrossing/README.md

```diff
@@ -0,0 +1,359 @@
+# VisualCrossing Binding
+
+Visual Crossing Weather is the easiest-to-use and lowest-cost source for historical and forecast weather data.
+Our [Weather API](https://www.visualcrossing.com/weather-api) is designed to integrate easily into any app or code, and
+our prices are lower than any other provider in the industry.
+
+Our data is used daily by a diverse customer-base including business analysts, data scientists, insurance professionals,
+energy producers, construction planners, and academics.
+
+We have the [Weather Data](https://www.visualcrossing.com/weather-data) and expertise needed to serve any individual or
+organization from an independent event planner to a global enterprise.
+
+## Supported Things
+
+- `weather`: [Weather API](https://www.visualcrossing.com/weather-api)
+
+## Thing Configuration
+
+_Describe what is needed to manually configure a thing, either through the UI or via a thing-file._
+_This should be mainly about its mandatory and optional configuration parameters._
+
+_Note that it is planned to generate some part of this based on the XML files
+within ```src/main/resources/OH-INF/thing``` of your binding._
+
+### `sample` Thing Configuration
+
+| Name     | Type | Description                                                                                                                                                                                                                                               | Default | Required | Advanced |
+|----------|------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|----------|----------|
+| password | text | API Key to connect to the cloud                                                                                                                                                                                                                           | N/A     | yes      | no       |
+| location | text | Is the address, partial address or latitude,longitude location for which to retrieve weather data. You can also use US ZIP Codes. If you would like to submit multiple locations in the same request, consider our Multiple Location Timeline Weather API | OH location | no       | no       |
```

Comment:
Can you align the table (markdowntable.com)
Also think this can be removed:
`
If you would like to submit multiple locations in the same request, consider our Multiple Location Timeline Weather API
`

Reply:
It is formatted and this part is removed. I think you were looking on old code 
![obraz](https://github.com/user-attachments/assets/918c955d-0249-4483-a55f-91df9fc06b36)

Reply:
Weird. Sorry for that.